include/rmm/mr/device/arena_memory_resource.hpp

```diff
@@ -15,17 +15,18 @@
  */
 #pragma once
 
+#include <spdlog/common.h>
 #include <rmm/detail/error.hpp>
 #include <rmm/mr/device/detail/arena.hpp>
 #include <rmm/mr/device/device_memory_resource.hpp>
+#include <sstream>
 
 #include <cuda_runtime_api.h>
 
 #include <map>
 #include <shared_mutex>
```

Comment:
Group headers by library, in order of "nearest" to "farthest" relative to RMM. Space between groups. clang-format will handle alphabetizing within groups.

```suggestion
#include <rmm/detail/error.hpp>
#include <rmm/mr/device/detail/arena.hpp>
#include <rmm/mr/device/device_memory_resource.hpp>

#include <cuda_runtime_api.h>

#include <spdlog/common.h>

#include <map>
#include <shared_mutex>
#include <sstream>
```